### ISSUE TITLE
Trouble retreviting info on some distros

### ISSUE THREAD

**Issue by Xmetalfanx** [tinyMediaManager/tinyMediaManager#311]
__What TMM version are you using?__
2.9.12 , though the issue is one I have seen on at LEAST back to 2.9.10 as well ,,,updating version to see if this was a bug that was already fixed

__release, pre-release, nightly, or directly from GitHub/branch?__
Release ... downloaded via the link on the homepage linked to in the Github readme 

__What is the actual behaviour?__
On  SOME distros (further info will be below) retreviing show data = no result under IMDb.com or THeTVDB.com ... the same version of TMM on another distro BASE (even same tarball download, so its not like the download on the offending distro is bad) the behaviour is 100% as expected ... same files in same "structure" ... episode data is not scraped and nor is the show data.

__What is the expected behaviour?__
Show/Episode data scraped as normal.

__Steps to reproduce:__
- first
 Select a Show (or episode of a show on the affected distro base) and right click and (depending on if its a show or episode, select the menu option to get show OR episode data ...the results come up as "not found" all the time on IMDB.com or TheTVDB.com 

__Additional__
Have you attached the Logfile from the day it happened?
hope the attached log file is the correct one 
[launcher.log](https://github.com/tinyMediaManager/tinyMediaManager/files/2087497/launcher.log)
any other log file (maybe in a subdirectory) would be provided if requested

Your Operating system? (win/mac/linux? version?)
I have a few different Linux installs (for fun) and just as an example on OpenSuse Tumbleweed (rolling release), there is no problem with the same exact tarball download + same files .... 

I have to check my other systems that have the same base, but the OS I noticed this problem on is KUBUNTU 18.04.   i AM NOT sure if it's an 18.04 base problem but I have different machines that have other 18.04 based distros I could check.

I am 100% guessing but maybe its the fact the version of Java is different in the new Long Term Support releases, and that changed something TMM was using?   Anyway if its a simple missing package that is not installed that should be, I am open to that idea as well ... i figured it was a temp problem with say those services (I noticed the issue, two days in a row) but then I booted into Tumbleweed like i said .. did the same steps and the shows/episode info for the same exact files in the same location were detected and "applied" instantly ... no problem

**Comment by Xmetalfanx** [comment]
oops sorry ... still having the same issue that only seems to be happening in Kubuntu 18.04 (I do not recall this happening in KDE Neon which at the time of me typing this is still 16.04 based) ... I just noticed I attached the WRONG log file 

Here is the one that was needed 
[tmm.log](https://github.com/tinyMediaManager/tinyMediaManager/files/2087533/tmm.log)

**Comment by mlaggner** [comment]
please use Java 8 along with tmm - that's the latest _stable_ Java (Ubuntu ships Java 10 with 18.04 IIRC)

**Comment by mlaggner** [comment]
alternatively you could search for the error `java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty` along with Ubuntu 18.04/Java 10.
As far as I could see there is a problem with the installed certificates.. maybe you can solve it yourself (but I cannot guarantee that the rest of tmm works with Java 10 - Oracle still destroys backwards with every new version)

**Comment by Xmetalfanx** [comment]
I still cant explain WHY this only seems to be an issue on Kubuntu and NOT other 18.04 based distros, but looking up that error I ran across a few posts and bottom line TMM is "working" on Kubuntu 18.04 now.  Some of the links I have been looking at were [Lauchpad Link 1 ](https://bugs.launchpad.net/ubuntu/+source/ca-certificates-java/+bug/1739631) and [Launchpad Link 2](https://bugs.launchpad.net/ubuntu/+source/ca-certificates-java/+bug/1769013) which basically led me to the "fix" though again ... i have NO CLUE why this only seems to effect Kubuntu 18.04 out of the distros I have tried based on 18.04.   

This is NOT a problem with TMM at all but a bug apparently that has already been fixed but just not backported into Bionic (codename for 18.04) yet.   I installed the ca-certificates-java package for 18.10 (pre-alpha atm) from [ca-certificates-java, from Ubuntu 18.10 repo](https://packages.ubuntu.com/cosmic/ca-certificates-java) and as soon as I tried TMM, it scraped the info right away. 

I just hope anyone else with this issue will run across this thread ... at least until Canonical backports that version of ca-certificates-java into the default 18.04 repos, and they will be able to "fix" the issue.
